portal-api/src/v1/validation/fields/country.js

```diff
@@ -1,6 +1,15 @@
-const { findOneCountry } = require('../../controllers/countries.controller');
+const { getCountry } = require('../../controllers/countries.controller');
 
-module.exports.countryIsDisabled = (code) => {
-  const { data: country } = findOneCountry(code);
-  return country && country.disabled;
+/**
+ * Retrieves the disabled status of a country based on its country code.
+ * @param {string} code - The country code.
+ * @returns {boolean} - The disabled status of the country.
```

Comment:
is this a promise of a boolean?

Reply:
Boolean.

Reply:
Are you sure about this? It's an async function (sorry for a Sunday comment, will review properly again tomorrow :) )?
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/85a14945-80b9-4a1c-8c77-7a9ae76842c6)